src/layouts/EditHomepage.jsx

```diff
@@ -816,6 +1020,9 @@ export default class EditHomepage extends Component {
               </>
             ))}
           </div>
+          <div className={editorStyles.pageEditorFooter}>
+            <button type="button" className={elementStyles.blue} onClick={this.savePage}>Save</button>
```

Comment:
"Save" button still saves even if errors are present - we need to modify the `savePage` method.